### DISCUSSION TITLE
Navigationtree shows blank page

### DISCUSSION THREAD

**Discussion by nibreh** [datenstrom/community#1] [Report a bug] [answered]
Hello Markseu - I'm playing with yellowCMS, again and again

Problems with snippet navigationtree (I don't know really what is it) : 
It's give me a blank page !

**Comment by markseuffert** [comment, marked as answer]
I think I found it, please update Yellow... and the snippet should work again. 
Thanks junibreh for testing and reporting bugs! :smile:
